kart/cli.py

```diff
@@ -403,7 +403,9 @@ def load_commands_from_args(args, skip_first_arg=True):
 def entrypoint():
     freeze_support()
     load_commands_from_args(sys.argv)
-    cli()
+    # Don't let helper mode mess up the usage-text, or the shell complete environment variables.
+    prog_name = "kart" if os.path.basename(sys.argv[0]) == "kart_cli" else None
```

Comment:
what's the `None` case here for? Would that handle a case where you created a shell alias/symlink to kart called `k` or something like that?

Reply:
Yes, it seems to correctly handle that case - if you rename or alias kart, you get usage info for the renamed kart.
Depending on why kart was renamed or aliased, this may or may not be desirable, but at least it's not like tab-completion where something breaks if we get it wrong.